mixins/itemHelper.ts

```diff
@@ -63,6 +67,53 @@ const itemHelper = Vue.extend({
       } else {
         return false;
       }
+    },
+    /**
+     * Generate a link to the item's details page route
+     *
+     * @param {BaseItemDto} item - The item used to generate the route
+     * @returns {string} A valid route to the item's details page
+     */
+    getItemDetailsLink(item: BaseItemDto): string {
+      let routeName: string;
+      let routeParams: Record<never, never>;
+
+      if (item.Type && validLibraryTypes.includes(item.Type)) {
+        routeName = 'library-viewId';
+        routeParams = { viewId: item.Id };
+      } else {
+        switch (item.Type) {
+          case 'Series':
```

Comment:
Right now they're not in the OpenAPI spec, as server uses class name for items, instead of an enum. We could maybe use an enum type when generating the openapi json server-side.